prov/gni/src/gnix_ep.c

```diff
@@ -1755,14 +1761,13 @@ static int __gnix_ep_cm_nic_prep(struct gnix_fid_domain *domain,
 	if (info->src_addr &&
 	    info->src_addrlen == sizeof(struct gnix_ep_name)) {
 		name = (struct gnix_ep_name *)info->src_addr;
-		if (name->name_type == GNIX_EPN_TYPE_BOUND) {
-			/* EP name includes user specified service/port */
```

Comment:
I thought it wasn't very informative in this location. I'll update it with more info.